### PR TITLE
Various small fixes

### PR DESCRIPTION
**Pull request by pali**

### CODE REVIEW

File: lib/DBD/MariaDB.pod [hunk 1]
```diff
@@ -250,9 +250,9 @@ for the socket than that built into the client.
 
 =item mariadb_ssl
 
-A true value turns on the C<CLIENT_SSL> flag when connecting to the MariaDB or
-MySQL server and enforce SSL encryption. A false value (which is default)
-disable SSL encryption with the MariaDB or MySQL server.
+A true value enforce SSL encryption when connecting to the MariaDB or MySQL
+server. A false value (which is default) disable SSL encryption with the MariaDB
+or MySQL server.
```
Comment (choroba) [inline lib/DBD/MariaDB.pod:255]:
```suggestion
A true value enforces SSL encryption when connecting to the MariaDB or MySQL
server. A false value (which is default) disables SSL encryption with the MariaDB
or MySQL server.
```

File: .appveyor.yml [hunk 2]
```diff
@@ -48,7 +48,7 @@ install:
   - cinst StrawberryPerl --version %perl% %x86% --allow-empty-checksums
   - refreshenv
   - perl -V
-  - perl -e "print qq(o conf urllist push http://www.cpan.org\no conf commit\n)" | cpan
+  - perl -e "print qq(o conf urllist push http://www.cpan.org\n)" | cpan
```
Comment (choroba) [inline .appveyor.yml:51]: I tried it on my machine and it doesn't seem to commit. Are you sure it commits always?

**Comment by choroba** [comment]
Another possible small fix:
```diff
diff --git a/dbdimp.c b/dbdimp.c
index eff6e13..bbe875e 100644
--- a/dbdimp.c
+++ b/dbdimp.c
@@ -2553,7 +2553,7 @@ static bool mariadb_db_my_login(pTHX_ SV* dbh, imp_dbh_t *imp_dbh)
   if (DBIc_TRACE_LEVEL(imp_xxh) >= 2)
     PerlIO_printf(DBIc_LOGPIO(imp_xxh),
                  "imp_dbh->mariadb_db_my_login : dbname = %s, uid = %s, pwd = %s,"
-                 "host = %s, port = %u\n",
+                 " host = %s, port = %u\n",
                  dbname ? dbname : "NULL",
                  user ? user : "NULL",
                  !password ? "NULL" : !password[0] ? "" : "****",
```

File: .appveyor.yml (hunk 2 above)
Comment (pali) [reply .appveyor.yml:51]: It looks like that on AppVeyor before this change is _commit_ called two times:
https://ci.appveyor.com/project/perl5-dbi/dbd-mariadb/builds/47523009/job/watsk1heaywf61ep#L88

And with this change is _commit_ called just once:
https://ci.appveyor.com/project/perl5-dbi/dbd-mariadb/builds/47523083/job/mjliptfb27d0lur3#L89

So I was in impression that `urllist push` must do implicit `commit`. Hm... maybe implicit commit is done after exiting from cpan command? Or AppVeyor cpan is different?

File: lib/DBD/MariaDB.pod (hunk 1 above)
Comment (pali) [reply lib/DBD/MariaDB.pod]: Fixed.

**Comment by pali** [comment]
> ```diff
> -                 "host = %s, port = %u\n",
> +                 " host = %s, port = %u\n",
> ```

Added new commit.

File: .appveyor.yml (hunk 2 above)
Comment (choroba) [reply .appveyor.yml:51]: Maybe it has
```
o conf auto_commit 1
```

Comment (pali) [reply .appveyor.yml:51]: Good catch, it looks like that StrawberryPerl has enabled cpan auto_commit by default:
https://github.com/StrawberryPerl/Perl-Dist-Strawberry/blob/master/share/config-files/CPAN_Config.pm.tt#L27

Comment (pali) [reply .appveyor.yml:51]: I updated commit message.

**Pull request merged**